credoai/prism/compare.py

```diff
@@ -0,0 +1,64 @@
+"""Performs comparison between 2 pipelines"""
+from typing import List, Literal, Optional
+from credoai.evaluators.utils.validation import check_instance
+from credoai.evidence.containers import MetricContainer
+from credoai.lens import Lens
+
+from credoai.utils import flatten_list
+from credoai.prism.comparators.metric_comparator import MetricComparator
+
+
+class Compare:
+    SUPPORTED_CONTAINERS = [MetricContainer]
+
+    def __init__(
+        self,
+        pipelines: List[Lens],
+        ref: Optional[str] = None,
+        operation: Literal["diff", "ratio"] = "diff",
+        abs: bool = False,
+    ):
+        """
+        Compare results between 2 different Lens runs.
+
+        Depending on the type of containers detected, this class will invoke
+        the appropriate comparator.
+
+        Parameters
+        ----------
+        results_primary : List[EvidenceContainer]
```

Comment:
These parameters don't match the initialization. Also the docstring is under __init__ rather than the class name